cityofaustin/atd-moped
# Create project tags ATSD

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13668

## Testing
**URL to test:** Local

**Steps to test:**
1. Star the cluster and editor.
2. Observe new tags in the data dictionary.
3. Apply the down migration

```shell
# ./moped-database
$  hasura migrate apply --down 1
``` 
4. Observe the tags are gone

---
#### Ship list
